sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/Models/DigitalTwinsSendTelemetryOptions.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.DigitalTwins.Core
+{
+    /// <inheritdoc />
+    [CodeGenModel("DigitalTwinsSendTelemetryOptions")]
```

Comment:
we have called this API `PublishTelemetry` shouldn't we rename the option to match the API name?